Rakefile

```diff
@@ -20,5 +20,7 @@ require "yast/rake"
 
 Yast::Tasks.configuration do |conf|
   # The package does not live in the official YaST:Head OBS project
-  conf.obs_project = "home:aschnell:storage-redesign"
+  conf.obs_project = "YaST:storage-ng"
+  # Non-existent project to prevent accidental submission to Tumbleweed
+  conf.obs_sr_project = "NONE"
```

Comment:
Our tasks are not prepared to skip the request in case of `nil`. In the best case, setting it to `nil` is equivalent to setting it to the default value (i.e. Factory). In the worst case is equivalent to setting it to the empty string... thus invalid osc commands.